### ISSUE TITLE
Can't click to set/clear breakpoint when a gutter decoration exists

### ISSUE THREAD

**Issue by eamodio** [microsoft/vscode#95992]
Here I am using the Bookmarks extension, and you can't click where the bookmark is to set/clear a break point.
![image](https://user-images.githubusercontent.com/641685/80131062-a7b4ef80-8567-11ea-858a-ab49a4997b8b.png)

If I hit F9 to set the break point it works as I would expect, even with the breakpoint indicator on top of the bookmark one.
![image](https://user-images.githubusercontent.com/641685/80131135-c74c1800-8567-11ea-8dfc-8e6cdeaae2b6.png)

**Comment by alefragnani** [comment]
Hi @eamodio ,

This may be _somehow related_ to #5923. I remember in the early days, only one of the decorators where displayed (don’t remember which one) but in recent releases, both are displayed, mixed. 

I think the gutter decorators (in this case) should receive similar animation you see in Tree Views, when you do filtering (displayed at its top-right corner). When you move the mouse over that filter, it expands with an additional clear button.

I hope fixing/updating this, would be one step further to deliver #5455 😬

**Comment by alefragnani** [comment]
Hi @eamodio ,

The 1.78 release provided some updates on this (https://code.visualstudio.com/updates/v1_78#_glyph-margin-decoration-rendering-improvements), so maybe your issue has been solved. 

For me, the Bookmarks are working almost perfectly, with a small issue that I reported here (https://github.com/microsoft/vscode/issues/181982). It's just a minor detail, and would probably be handled in upcoming releases. I suppose the hardest part has been solved in 1.78, which is great.

Hope this helps

**Comment by eamodio** [comment]
@alefragnani Yeah, except for the issue tracked in #181982 it looks like this is resolved.

**Comment by hyangah** [comment]
It looks like we can clear breakpoints, but cannot set new breakpoints on the lines covered with gutter deco. 
Is https://github.com/microsoft/vscode/issues/185814 the blocking task, or is it unrelated?
